src/Action/TaskList/SetItems.php

```diff
@@ -17,45 +17,27 @@
 final class SetItems
 {
     public function __construct(
-        private EntityManagerInterface $objectManager,
+        private EntityManagerInterface $entityManager,
         private UserManager $userManager,
         private TaskListManager $taskListManager,
         private TaskListNormalizer $taskListNormalizer
     )
-    {
-    }
-
-    private function getTaskList(\DateTime $date, User $user)
-    {
-        $taskList = $this->objectManager
-            ->getRepository(TaskList::class)
-            ->findOneBy(['date' => $date, 'courier' => $user]);
-
-        if (null === $taskList) {
-            $taskList = new TaskList();
-            $taskList->setDate($date);
-            $taskList->setCourier($user);
-            $this->objectManager->persist($taskList);
-            $this->objectManager->flush();
-        }
-
-        return $taskList;
-    }
+    {}
 
     public function __invoke(Request $request)
     {
         $date = new \DateTime($request->get('date'));
         $user = $this->userManager->findUserByUsername($request->get('username'));
 
-        $taskList = $this->getTaskList($date, $user);
+        $taskList = $this->taskListManager->getTaskListForUser($date, $user);
```

Comment:
~Maybe the `$date` should be the date of the delivery ?~

Reply:
~Like what happen if i create a delivery for the next week ?~

Reply:
mmm it is an unrelated change sorry
here we are in the endpoint that assigns task when on the dispatch panel so the $date is a parameter we passed (the date we are showing in the dispatch)